components/atom/icon/src/index.scss

```diff
@@ -16,7 +16,8 @@ $atom-icon-colors: (
   'error': $c-error,
   'primary': $c-primary,
   'success': $c-success,
-  'gray': $c-gray
+  'gray': $c-gray,
+  'grayLight': $c-gray-light-1
```

Comment:
Do you think we should have a detailed key for gray light 1 variation? Like `grayLight1` instead of have `grayLight` with gray light 1 variation.